guides/release/applications/dependency-injection.md

```diff
@@ -252,11 +254,11 @@ export default Component.extend({
     let audioType = this.song.audioType;
 
     return applicationInstance.lookup(`service:audio-${audioType}`);
-  }),
+  };
```

Comment:
I find it fairly odd to have semicolons here (I had never seen this before in class bodies until recently), I would normally not put anything here (no semicolon, no comma, just a new line).

I don't think this is a giant issue, but we do need to ensure that the guides consistently follow one pattern or the other (either semicolons or not).

Reply:
We're using a mix everywhere now :( I'll make a note to catch it in an editing pass.